tests/test_query_scans.py

```diff
@@ -1,9 +1,9 @@
 import json
-import httpretty
```

Comment:
Why change the file location of all tests from `w3af_api_client/tests/` to `tests`?

Reply:
I prefer to keep tests separate from actual application code. This has the advantage that your tests can run against an installed version.